Source/TextKit/ASTextKitCoreTextAdditions.m

```diff
@@ -97,8 +98,13 @@ BOOL ASAttributeWithNameIsUnsupportedCoreTextAttribute(NSString *attributeName)
       cleanAttributes[NSForegroundColorAttributeName] = [UIColor colorWithCGColor:(CGColorRef)coreTextValue];
     }
     // kCTParagraphStyleAttributeName -> NSParagraphStyleAttributeName
-    else if ([coreTextKey isEqualToString:(NSString *)kCTParagraphStyleAttributeName] && ![coreTextValue isKindOfClass:[NSParagraphStyle class]]) {
-      cleanAttributes[NSParagraphStyleAttributeName] = [NSParagraphStyle paragraphStyleWithCTParagraphStyle:(CTParagraphStyleRef)coreTextValue];
+    else if ([coreTextKey isEqualToString:(NSString *)kCTParagraphStyleAttributeName]) {
+      if ([coreTextValue isKindOfClass:[NSParagraphStyle class]]) {
+        cleanAttributes[NSParagraphStyleAttributeName] = (NSParagraphStyle *)coreTextValue;
+      }
+      else {
```

Comment:
Please adjust if-else style to match project; else { should be on the line with the }.

Reply:
Hmm, I guess other parts of this file are not conforming either. So this is probably OK.